hazelcast-client-new/src/main/java/com/hazelcast/client/proxy/ClientMapReduceProxy.java

```diff
@@ -269,25 +261,10 @@ public boolean cancel(boolean mayInterruptIfRunning) {
         }
 
         @Override
-        public boolean isCancelled() {
-            return cancelled;
-        }
-
-        @Override
-        public void setResult(Object result) {
+        protected void setResult(Object result) {
             super.setResult(result);
-            latch.countDown();
```

Comment:
Unfortunately it's used in line 120 and 133, so I can't get rid of it completely